### DISCUSSION TITLE
License Change Request

### DISCUSSION THREAD

**Discussion by undefined-moe** [hydro-dev/Hydro#138] [General]
We are planning to change license of this repo to restrict commercial usage, and we need contributor's approval.
我们正在计划修改开源许可证以对商用作出限制。并正在征得项目贡献者同意。

@Macesuted @wuxianucw @junukwon7 @Rratic @AsakuraMizu @KunoiSayami 

You can reply to this thread directly. (Approve/Reject [reason])
请在下方直接回复 （同意/不同意[理由])。

**Comment by KunoiSayami** [comment]
Approved.

**Comment by junukwon7** [comment]
Approved

**Comment by Macesuted** [comment]
Approved.

**Comment by wuxianucw** [comment]
Approve.

**Comment by Rratic** [comment]
这啥时候的
不是应该忽视我吗
Approve

**Comment by AsakuraMizu** [comment]
啥玩意咋还带我玩

Approved

**Comment by BATTLEHAWK00** [comment]
Approved

**Comment by Nerlci** [comment]
Approved.

**Comment by undefined-moe** [comment]
草不是这个thread
直接reply到pr上就行

**Comment by junukwon7** [comment]
Approved
